Tests/FeedStoreChallengeTests.swift

```diff
@@ -3,7 +3,7 @@
 //
 
 import XCTest
-import FeedStoreChallenge
+@testable import FeedStoreChallenge
```

Comment:
No need for `@testable` - you can remove it.

```suggestion
import FeedStoreChallenge
```